e2e/testdata/fn-eval/exec-function/.expected/diff.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/resources.yaml b/resources.yaml
+index e8ae6bb..953c080 100644
+--- a/resources.yaml
++++ b/resources.yaml
+@@ -1,4 +1,3 @@
+-# Copyright 2021 Google LLC
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+@@ -11,11 +10,11 @@
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-apiVersion: apps/v1
++apiVersion: apps/v1 # Copyright 2021 Google LLC
```

Comment:
Once you rebase after the `bumpup-go-yaml` PR is merged, the license header related diff will go away.